src/charm.py

```diff
@@ -154,11 +156,31 @@ def __init__(self, *args, **kargs) -> None:
             self.unit.status = ops.BlockedStatus(exc.msg)
             return
 
+        self._event_timer = EventTimer(self.unit.name)
+        try:
+            reconcile_evt_timer_is_active = self._event_timer.is_active(RECONCILE_RUNNERS_EVENT)
```

Comment:
Can `reconcile_evt_timer_is_active` be rename to `reconcile_event_time_is_active` or `reconcile_time_is_active`? Or something like that?
I feel `evt` a common shorthand for `event`.